.eslintrc

```diff
@@ -5,11 +5,25 @@
         "prettier",
         "prettier/react"
     ],
+    "parser": "@typescript-eslint/parser",
     "plugins": [
         "prettier"
     ],
     "env": {
         "es6": true,
         "browser": true
+    },
+    "rules": {
+        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx", ".tsx"] }],
+        "jsx-a11y/no-static-element-interactions": 1,
```

Comment:
these accessibility rules were reporting as errors and were pissing me off. we can deal with them later.